src/Cloud/weglide/WeGlideSettings.hpp

```diff
@@ -25,10 +25,20 @@ Copyright_License {
 #define XCSOAR_CLOUD_WEGLIDE_WEGLIDESETTINGS_HPP
 
 #include "time/BrokenDate.hpp"
+#include "util/StaticString.hxx"
 
 #include <cstdint>
 
-/**
+namespace WeGlide {
+struct Pilot {
+  uint32_t id;
+  BrokenDate birthdate;
+  StaticString<0x40> name;
```

Comment:
I'm confused by your commit message. 
"WeGlide Cloud/weglide/WeGlideSettings Update" - what does that mean?
This commit does two things:
1. move stuff to `struct Pilot`
2. add the `name` field
Doing two different things in one commit is confusing enough, but this commit message certainly doesn't help explain the confusion.

Reply:
Ok I will split it in 2 commits

Reply:
... no, I removed this commit, want to do it later (after this intial PR)